src/app/pages/credentials/new-users/all-users/users-search/users-search.component.ts

```diff
@@ -0,0 +1,217 @@
+import { SearchProperty } from 'app/modules/forms/search-input/types/search-property.interface';
+import { AdvancedSearchQuery, SearchQuery } from 'app/modules/forms/search-input/types/search-query.interface';
+import { booleanProperty, searchProperties, textProperty } from 'app/modules/forms/search-input/utils/search-properties.utils';
+import { ApiDataProvider } from 'app/modules/ix-table/classes/api-data-provider/api-data-provider';
+import { FakeProgressBarComponent } from 'app/modules/loader/components/fake-progress-bar/fake-progress-bar.component';
+import { TestDirective } from 'app/modules/test-id/test.directive';
+import { UrlOptionsService } from 'app/services/url-options.service';
+
+@UntilDestroy()
+@Component({
+  selector: 'ix-users-search',
+  templateUrl: './users-search.component.html',
+  styleUrl: './users-search.component.scss',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  standalone: true,
+  imports: [
+    FakeProgressBarComponent,
+    MatButton,
+    AsyncPipe,
+    SearchInputComponent,
+    TranslateModule,
+    TestDirective,
+  ],
+})
+export class UsersSearchComponent implements OnInit {
+  readonly userName = injectParams('id');
+  protected readonly advancedSearchPlaceholder = this.translate.instant('Username = "root" AND Builtin = "Yes"');
+
+  dataProvider = input.required<ApiDataProvider<'user.query'>>();
+
+  protected searchQuery = signal<SearchQuery<User>>({
+    query: '',
+    isBasicQuery: true,
+  });
+
+  protected readonly searchProperties = signal<SearchProperty<User>[]>([]);
+
+  constructor(
+    private urlOptionsService: UrlOptionsService,
+    private translate: TranslateService,
+    private activatedRoute: ActivatedRoute,
+  ) { }
+
+  ngOnInit(): void {
+    this.loadParamsFromRoute();
+
+    this.dataProvider().sortingOrPaginationUpdate
+      .pipe(untilDestroyed(this))
+      .subscribe(() => {
+        this.updateUrlOptions();
+      });
+
+    this.dataProvider().currentPage$.pipe(
+      filter(Boolean),
+      untilDestroyed(this),
+    ).subscribe({
+      next: (users: User[]) => {
+        this.setSearchProperties(users);
+      },
+    });
+  }
+
+  private setSearchProperties(users: User[]): void {
+    const groups = new Set<string>();
+    for (const user of users) {
+      groups.add(user.group.id.toString());
+      if (!user.groups) {
+        continue;
+      }
+      for (const group of user.groups) {
+        groups.add(group.toString());
+      }
+    }
+
+    const roles = new Set<Role>();
+    for (const user of users) {
+      for (const role of user.roles) {
+        roles.add(role);
+      }
+    }
+    this.searchProperties.update(() => searchProperties<User>([
+      textProperty('id', this.translate.instant('ID'), of([] as Option[])),
+      textProperty('username', this.translate.instant('Username'), of([] as Option[])),
+      textProperty(
+        'fullname',
```

Comment:
![image](https://github.com/user-attachments/assets/84c42e6b-b16a-4eb8-a4cb-67adb947d2e3)

No search results for `Full Name` property or details panel displays wrong information.